src/model/enhance-interactive-content-elements.ts

```diff
@@ -0,0 +1,81 @@
+import { stripHTML } from '@root/src/model/strip-html';
+
+const isInteractiveContentBlockElement = (element: CAPIElement): boolean =>
+	element._type ===
+		'model.dotcomrendering.pageElements.InteractiveBlockElement' &&
+	element?.scriptUrl === 'https://uploads.guim.co.uk/2019/03/20/boot.js';
```

Comment:
I think the script element can also be either

- `https://interactive.guim.co.uk/page-enhancers/nav/boot.js` or
- `https://uploads.guim.co.uk/2019/12/11/boot.js` or
- `https://uploads.guim.co.uk/2019/03/20/boot.js`

I'll forward email from CP as an FYI

Reply:
👍 Thank you! I almost created a `scriptUrls` array when creating the function and then was like, nah, there's probably just this one 🤣